.github/workflows/main.yml

```diff
@@ -43,4 +45,21 @@ jobs:
         run: mix deps.compile
 
       - name: Run tests
-        run: mix test.integration --trace
+        run: mix test --trace
+
+      - name: Install protoc
+        if: ${{matrix.integration}}
+        run: |
+          wget https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
+          unzip protoc-3.12.3-linux-x86_64.zip
```

Comment:
Even though it doesn't take too long, it would be ideal if we could cache this. Maybe an improvement for the next iteration.